src/data/pokemon/species_info.h

```diff
@@ -41614,10 +41821,10 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .evYield_Defense = 2,
         .evYield_SpDefense = 1,
         .description = COMPOUND_STRING(
-            "Generations of kings were attended to\n"
-            "by these Pokémon, which used their\n"
-            "spectral power to manipulate and control\n"
-            "people and Pokémon."),
+            "In this defensive stance,\n"
+            "Aegislash uses its steel body and a force\n"
+            "field of spectral power to reduce the\n"
+            "damage of any attack."),
```

Comment:
Why was it changed to the Sword/Shield description instead of keeping X/Y?

Reply:
Because XY doesn't distinguish between the two forms, see also https://discord.com/channels/419213663107416084/774393519569502268/1184117739502182501